src/main/java/dukepro/handlers/TasksManager.java

```diff
@@ -0,0 +1,152 @@
+package dukepro.handlers;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import dukepro.tasks.Task;
+
+/**
+ * Class for TasksManager.
+ */
+public class TasksManager {
+    private static ArrayList<Task> tasks = new ArrayList<>();
```

Comment:
Interesting to initialise the arraylist here, perhaps you could explain why you choose to initialise the arraylist here instead of the constructor?